README.md

```diff
@@ -16,25 +18,38 @@ curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-
 (Invoke-WebRequest -Uri https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py -UseBasicParsing).Content | python -
 ```
 
+## Prerequisite Setup
+
+We use [Trello's](https://trello.com/) API to save and fetch to-do tasks. In order to call their API, you need to first 
+[create an account](https://trello.com/signup), then generate an API key and token by following the 
+[instructions here](https://trello.com/app-key).
+
 ## Dependencies
 
-The project uses a virtual environment to isolate package dependencies. To create the virtual environment and install required packages, run the following from your preferred shell:
+The project uses a virtual environment to isolate package dependencies. To create the virtual environment and install 
+required packages, run the following from your preferred shell:
 
 ```bash
 $ poetry install
 ```
 
-You'll also need to clone a new `.env` file from the `.env.template` to store local configuration options. This is a one-time operation on first setup:
+You'll also need to clone a new `.env` file from the `.env.template` to store local configuration options. This is a 
+one-time operation on first setup:
 
 ```bash
 $ cp .env.template .env  # (first time only)
 ```
 
-The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
+The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like 
+development mode (which also enables features like hot reloading when you make a file change).
+
+The `.env` file also needs updating with the Trello API key and token created earlier, along with the board ID you are
```

Comment:
Nice clear instructions 😄 
Only thing that might be missing is that your code relies on the board having headings "Not Started", "In Progress" & "Completed" which might be worth explicitly calling out